configs/eln_extras_meta.yaml

```diff
@@ -27,6 +27,7 @@ data:
     - duo_unix
     - duo_unix-selinux
     - ebranch
+    - emacs-nw
```

Comment:
emacs-nw is in ELN.

Reply:
Is it actually shipped though? `mock -r fedora-eln-x86_64 --install emacs-nw` fails for me here.

Reply:
I see it listed in https://github.com/minimization/content-resolver-input/blob/main/configs/sst_cs_plumbers-editors.yaml so in theory it should be (and https://tiny.distro.builders/workload--sst_cs_plumbers-editors--fedora-empty-base--repository-fedora-eln--x86_64.html confirms it), but I don't see in https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/AppStream/x86_64/os/Packages/ .

Reply:
That would need to be fixed in https://pagure.io/pungi-fedora/tree/eln , not in content-resolver.

Reply:
Pungi is just installing the emacs group, so I think the actual fix for this needs to be in comps: https://pagure.io/fedora-comps/pull-request/950